build.gradle

```diff
@@ -646,6 +659,16 @@ task functional(type: Test) {
     include "uk/gov/hmcts/ccd/datastore/tests/functional/**"
     include "uk/gov/hmcts/ccd/datastore/tests/v2/**"
     useJUnitPlatform()
+
```

Comment:
some explanatory comments here would be good too. I know we don't have many comments in the build gradle but we should start getting in the habit to add them.